index.js

```diff
@@ -163,7 +163,8 @@ class ExecutorQueue extends Executor {
      * @return {Promise}
      */
     async postBuildEvent({ pipeline, job, apiUri }) {
-        const jwt = this.tokenGen(Object.keys(pipeline.admins)[0], {}, pipeline.scmContext);
+        const admin = await pipeline.admin;
+        const jwt = this.tokenGen(admin.username, {}, pipeline.scmContext);
```

Comment:
don't think there is username, it's just `admin` based on the logic here https://github.com/screwdriver-cd/models/pull/247/files